app/views/layouts/angular/_gtl.html.haml

```diff
@@ -0,0 +1,37 @@
+- current_model = controller.class.model if controller.class.respond_to?("model") && !controller.class.model.nil? rescue nil
```

Comment:
is this correct way? The problem here is that model can be obtained either from current controller or from ApplicationController. However if no model exists for `SomeController` there was an error with `uninitialized constant MiqAeCustomization` and continue with model being nil.